server/authentication/auth.ts

```diff
@@ -40,9 +40,8 @@ function init(): void {
       customHeaders: { Authorization: generateOauthClientToken() },
     },
     (token, refreshToken, params, profile, done) => {
-      console.log(token)
```

Comment:
Need to make sure this is removed in your current branch - don't want to be logging out user tokens anywhere

Reply:
Yes will do Jon.